lib/compiler-cranelift/src/translator/code_translator.rs

```diff
@@ -1291,15 +1381,27 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             )?;
             let splatted = builder.ins().splat(type_of(op), state.pop1());
             state.push1(splatted)
+
+            // let opcode = ir::Opcode::LoadSplat;
```

Comment:
commented out code again

Reply:
It will be useful for the future (is directly ported from cranelift-wasm, but can't be used just yet since it requires for Cranelift to publish a new version), that's why is useful keeping it!